docs/logs/README.md

```diff
@@ -153,3 +153,22 @@ instances if they are created by you.
 category name. Refer to the [.NET official
 document](https://learn.microsoft.com/dotnet/core/extensions/logging#log-category)
 to learn more.
+
+## Log Correlation
+
+In OpenTelemetry, logs are automatically correlated to traces. Check the [Log
+Correlation](./correlation/README.md) tutorial to learn more.
+
+## Log Enrichment
+
+TBD
+
+## Log Filtering
+
+TBD
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/customizing-the-sdk#log-filtering
We can link to this to achieve the current basic filtering based on severity/category.

Reply:
Updated, thanks!